docs/reference/create-client.txt

```diff
@@ -694,6 +694,20 @@ Ruby Options
      - ``Float``
      - none
 
+   * - ``:srv_max_hosts``
+     - The maximum number of mongos connections that may be created
+       for sharded topologies. If this option is set to 0, there will
```

Comment:
I would adjust the first sentence to something like "The maximum number of mongoses that the driver will communicate with", because the number of connections is controlled by pool size and can exceed 1 per mongos.